samples/no-editor.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: workspace.che.eclipse.org/v1alpha1
+kind: Workspace
+metadata:
+  name: cloud-shell
```

Comment:
is it really cloud-shell? Will controller propagate a default editor?

Reply:
My bad, these got added half-accidentally as I threw them together for testing. I'll update the names since they could still be useful for test cases.